extensions/gc/tasks/GarbageCollectorTask.js

```diff
@@ -17,6 +17,11 @@ class GarbageCollectorTask extends BackbeatTask {
         super();
         const gcState = gc.getStateVars();
         Object.assign(this, gcState);
+        const retryParams =
+        gcState.gcConfig?.consumer.retry;
+        if (retryParams) {
+            Object.assign(this, retryParams);
```

Comment:
Not sure this will work. I think `Object.assign` will put the fields/values that are inside `retryParams` and add then to `this` instead of adding `retryParams` as a field. So instead of having
```
{
    retryParams: {
      "backoff": {
        "factor": "1.5",
        "jitter": "0.1",
        "max": 300000,
        "min": 1000
      },
      "maxRetries": 5,
      "timeoutS": 300
    }
}
```
we get
```
{
    "backoff": {
        "factor": "1.5",
        "jitter": "0.1",
        "max": 300000,
        "min": 1000
    },
    "maxRetries": 5,
    "timeoutS": 300
}
```

Reply:
Indeed fixed here : https://github.com/scality/backbeat/pull/2520/commits/c80b93d63bf745e80314515d17a6ad8aa74347cf#diff-7beaf0ffcd262dad5a78c1b25cb7d54b9ac577441b372167f96d8c182be8fcf7R22